backend/clickhouse/query.go

```diff
@@ -961,3 +944,36 @@ func repr(val reflect.Value) string {
 		return val.String()
 	}
 }
+
+func getSortOrders[TReservedKey ~string](
+	sb *sqlbuilder.SelectBuilder,
+	paginationDirection modelInputs.SortDirection,
+	config model.TableConfig[TReservedKey],
+	params modelInputs.QueryInput,
+) (string, string) {
+	sortColumn := "timestamp"
+	sortDirection := modelInputs.SortDirectionDesc
+	if params.Sort != nil {
+		sortColumn = params.Sort.Column
+		sortDirection = params.Sort.Direction
+	}
+
+	if col, found := config.KeysToColumns[TReservedKey(sortColumn)]; found {
+		sortColumn = col
+	} else {
+		sortColumn = fmt.Sprintf("%s[%s]", config.AttributesColumn, sb.Var(sortColumn))
```

Comment:
@mayberryzane I think this is the only place we ended up needed to parameterize since we would know it's a mapped column if we hit the other side of this condition. Trying to wrap the string in `Var` again also seemed to break SQL building.